<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>801</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1217</width>
    <height>801</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1217</width>
    <height>801</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="left_frm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>781</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="data_graph_frm">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>510</y>
       <width>991</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="data_spike_frm">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>721</width>
       <height>521</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="text_left">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>0</y>
       <width>261</width>
       <height>501</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7.8pt;&quot;&gt;text&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="right_frm">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>231</width>
      <height>781</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="text_right">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>501</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select a file&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QFrame" name="btns_frm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="open_btn">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>OPEN FILE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="test_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TEST GRAPH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="test_button2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TEST GRAPH 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="xbtn_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
